platform-tooling-support-tests/platform-tooling-support-tests.gradle.kts

```diff
@@ -69,10 +69,10 @@ tasks.test {
 		// systemProperty("junit.jupiter.execution.parallel.enabled", "true")
 
 		// Pass version constants (declared in Versions.kt) to tests as system properties
-		systemProperty("Versions.apiGuardian", Versions.apiGuardian)
-		systemProperty("Versions.assertJ", Versions.assertJ)
-		systemProperty("Versions.junit4", Versions.junit4)
-		systemProperty("Versions.ota4j", Versions.ota4j)
+		systemProperty("Versions.apiGuardian", versions.apiguardian)
+		systemProperty("Versions.assertJ", versions["assertj"])
+		systemProperty("Versions.junit4", versions.junit4)
+		systemProperty("Versions.ota4j", versions.opentest4j)
```

Comment:
Should all 4 lines follow the `versions["assertj"]` pattern?

Reply:
See rationale in https://github.com/junit-team/junit5/pull/2236#issuecomment-609015745. But I can roll that last commit back if the rest of the team feels strongly about it.

Reply:
Understood. Let's keep them as they are for the time being.